provides.py

```diff
@@ -1,127 +1,247 @@
-import json
+from charms.reactive import Endpoint
+from charms.reactive import when, when_not
+from charms.reactive import set_flag, clear_flag, toggle_flag
 
-from charms.reactive import hook
-from charms.reactive import scopes
-from charms.reactive import RelationBase
+from tls_certificates_common import CertificateRequest
```

Comment:
This form of the import was needed to make the doc building work but is causing import errors when in use.